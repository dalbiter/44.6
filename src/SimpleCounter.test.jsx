import { render, screen } from '@testing-library/react';
import SimpleCounter from './SimpleCounter';
import { it } from 'vitest';


//smoke test
it('renders SimpleCounter component', () => {
    render(<SimpleCounter />)
});

//snapshot test
it('should match snapshot', () => {
    const { asFragment } = render(<SimpleCounter />)
    expect(asFragment()).toMatchSnapshot()
});