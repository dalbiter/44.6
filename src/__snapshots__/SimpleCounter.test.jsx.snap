// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Count: 0
    </h3>
    <button>
      Up
    </button>
    <button>
      Up By 2
    </button>
  </div>
</DocumentFragment>
`;
